config/routes.rb

```diff
@@ -1,16 +1,27 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
   devise_for :users
```

Comment:
- [ ] Nice working on your project, kindly check I get the error in the screenshot below when trying to sign in.
![image](https://user-images.githubusercontent.com/48371156/204101607-f77d43ce-8643-4321-80cf-3b2216afe54c.png)